app/models/application_qualification.rb

```diff
@@ -16,4 +25,16 @@ class ApplicationQualification < ApplicationRecord
   def missing_qualification?
     qualification_type == 'missing'
   end
+
+  def incomplete_degree_information?
+    return false unless degree?
+
+    EXPECTED_DEGREE_FIELDS.each do |field|
+      if send(field).nil?
```

Comment:
Could any of these contain empty strings? If so would that mean it's incomplete?

Reply:
Can this be shortened to something like: `EXPECTED_DEGREE_FIELDS.any? { |field| send(field).nil? }`

Reply:
@stevehook 
Empty strings are unlikely given the presence validations across all the forms, but should be considered incomplete.

Have addressed both questions in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2299/commits/dd499b41d4a7da192fdabe39cfb1e12c778a4b99

WDYT?

Reply:
Looks good :+1: